sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -588,6 +591,9 @@ class PeekedMessage(Message):
 
     def __init__(self, message):
         # type: (uamqp.message.Message) -> None
```

Comment:
@annatisch quick guidelines question:  If we're conforming to the "all objects must accept dict representation" guideline, should we express this at the type level even though it's very much "not the preferred flow"?  I'm very torn here between being "completely honest" and "providing less confusing guidance and relying on mypy+dict coercing users to express intent" (in terms of muddying up all the parameter types with Union[dict, ...])

Reply:
for posterity, from Anna:
"I would say don't add it for now - we generally don't across our SDKs at this point. If we decide we want to explicitly publicize the dict support, it's easy to add later"